polygon/heimdall/range_index.go

```diff
@@ -0,0 +1,83 @@
+package heimdall
+
+import (
+	"context"
+	"encoding/binary"
+
+	"github.com/c2h5oh/datasize"
+	"github.com/ledgerwatch/log/v3"
+
+	"github.com/ledgerwatch/erigon-lib/kv"
+	"github.com/ledgerwatch/erigon-lib/kv/mdbx"
+)
+
+type RangeIndex struct {
+	db kv.RwDB
+}
+
+const rangeIndexTableName = "Index"
+
+func NewRangeIndex(tmpDir string, logger log.Logger) (*RangeIndex, error) {
+	db, err := mdbx.NewMDBX(logger).
+		InMem(tmpDir).
+		Label(kv.SentryDB).
```

Comment:
if it's new database type - plz add new label

Reply:
:+1: